main.js

```diff
@@ -4,16 +4,30 @@ import Add from './modules/add.js';
 
 const add = new Add();
 const remove = new Remove();
-const createNew = document.getElementById('btn');
-const bookContainer = document.getElementById('book-list');
+// eslint-disable-next-line
+const { DateTime } = luxon;
+const now = DateTime.now();
+const mainPage = document.getElementById('main-page');
+const createBook = document.getElementById('add-book');
+const contactPage = document.getElementById('contact-page');
+const bookContainer = document.createElement('ul');
+bookContainer.classList += 'book-list';
+const heading = document.querySelector('h1');
+const mainContainer = document.querySelector('main');
+const time = document.querySelector('#current-time');
+// const bookContainer = document.getElementById('book-list');
```

Comment:
- [ ] Kindly avoid unnecessary commented code. It makes your code less readable. :pray: